src/Components/Profile.css

```diff
@@ -8,3 +8,12 @@
 .p-avatar-image > img {
   font-size: 0.5em;
 }
+
+@media screen and (max-width: 768px) {
```

Comment:
This can be done with built in prime react classes
https://www.primefaces.org/primeflex/width
https://www.primefaces.org/primeflex/lineheight
https://www.primefaces.org/primeflex/textalign

Reply:
I just noted that we can't change all properties within breakpoints with prime flex version 2.

Reply:
so will keep this media query then?

Reply:
Yea keep it. Its fine.